client/src/components/MessageList/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './Style.module.css';
+
+const data = [
+  {
+    self: true,
+    body: 'Hey Alice'
+  },
+  {
+    self: false,
+    body: 'Hi Bob'
+  }
+];
+
+var check1 = 0,
+  check2 = 0;
+
+const MessageList = (props) => {
+  return (
+    <div>
+      {data.map((data, index) => (
```

Comment:
Bad: ```data.map((data, index)``` - will introduce bug and confusion in no time.
Good: 
```data.map((msg, index)``` 
or
```data.map(({body, self}, index)```  

Most probably `self` is a reserved keyword. Let's change that to something like - `owner: true`